packer/configure-windows-vhd.ps1

```diff
@@ -56,6 +56,7 @@ function Get-FilesToCacheOnVHD
     $map = @{
         "c:\akse-cache\" = @(
             "https://github.com/Microsoft/SDN/raw/master/Kubernetes/flannel/l2bridge/cni/win-bridge.exe",
+            "https://github.com/microsoft/SDN/raw/master/Kubernetes/windows/debug/collectlogs.ps1",

```

Comment:
If we want this to work offline it might be good to get the other files too
https://github.com/microsoft/SDN/blob/42d0b1949387b699d4db1aea70dd20e71c0c6640/Kubernetes/windows/debug/collectlogs.ps1#L14

Reply:
https://github.com/microsoft/SDN/blob/42d0b1949387b699d4db1aea70dd20e71c0c6640/Kubernetes/windows/helper.psm1#L55

Reply:
ahh, missed that in helper.psm1
let me update this to add them

Reply:
@marosset : Collect log file expect all the files needs to be in c:\k\debug folder. 
We need to make change in the extract flow to put these files in above folder : https://github.com/microsoft/SDN/blob/42d0b1949387b699d4db1aea70dd20e71c0c6640/Kubernetes/windows/debug/collectlogs.ps1#L11

Reply:
@rjaini We'll move the files from akse-cache to c:\k\debug in the CSE so non-aks VHDs can put up the changes as well.
I'll make those changes after these changes get into the VHD to not break the air-gap scenario